app/src/pages/common/settingsPage/generalTab/generalTab.jsx

```diff
@@ -302,7 +302,9 @@ export class GeneralTab extends Component {
         <form onSubmit={this.props.handleSubmit(this.onFormSubmit)}>
           <div>
             <div className={cx('fake-input-label')}>Name</div>
-            <div className={cx('fake-input')}>{this.props.projectId}</div>
+            <div className={cx('fake-input')}>
+              <div className={cx('fake-input-text')}>{this.props.projectId}</div>
+            </div>
```

Comment:
What the reason wrap text with one more `div` if you can do the same thing without creating nesting?
Please move properties for truncing to `.fake-input` class, remove width and manage width with a paddings
Also I suggest to communicate with design team as it strange that we have different left\right paddings looks like mistake

Reply:
@chivekrodis you correct!